### PR TITLE
tiny fix: format code and tfix

### PR DESCRIPTION
**Pull request by Mashimiao**
Signed-off-by: Ma Shimiao <mashimiao.fnst@cn.fujitsu.com>

### CODE REVIEW

**Comment by q384566678** [comment]
need rebase.

**Comment by Mashimiao** [comment]
rebased, PTAL

**Comment by q384566678** [comment]
LGTM

**Comment by coolljt0725** [comment]
LGTM

**Comment by q384566678** [comment]
@caniszczyk Can you help me with the permissions problem? Thanks.

File: image/manifest_test.go [hunk 1]
```diff
@@ -225,6 +229,6 @@ func TestUnpackLayerRemovePartialyUnpackedFile(t *testing.T) {
 		t.Fatal(err)
 	}
 	if err == nil {
-		t.Fatal("Execpt partialy unpacked file has been removed")
+		t.Fatal("Execpt partially unpacked file has been removed")
```
Comment (jonboulle) [inline image/manifest_test.go]: expect

**Comment by q384566678** [comment]
> @caniszczyk Can you help me with the permissions problem? Thanks.

ping @caniszczyk.

**Comment by caniszczyk** [comment]
@q384566678 try again, looks like PullApprove didn't pickup the latest version of its config so I refreshed things

**Comment by q384566678** [comment]
LGTM

**Comment by q384566678** [comment]
@caniszczyk All right, thanks.

**Comment by Mashimiao** [comment]
Rebased, PTAL

**Comment by q384566678** [comment]
LGTM

**Comment by xiekeyang** [comment]
LGTM

**Pull request merged**